contributing/Code/How do I add a new field to the Task class.md

```diff
@@ -10,6 +10,8 @@ For example, It's fine to have a first release of a feature without `sort by` an
 
 "Sorting and grouping by blah is not yet supported. We are tracking this in [issue #nnnn](https://github.com/obsidian-tasks-group/obsidian-tasks/issues/nnn)."
 
+As soon as the field is added to the Task the SQL engine will be able to use it for filtering, sorting, grouping, etc. If you can add an example to the docs it will allow people to start to use and test its functionality.  
```

Comment:
I think any new field should be added to the `SQL Columns` page, though?

Reply:
I did a quick test of ORDER BY:

ORDER BY dueDate ASC, description ASC

worked.